radixdlt-java/src/main/resources/universe/betanet.json

```diff
@@ -48,6 +48,7 @@
                         "spin": 1,
                         "serializer": "radix.spun_particle",
                         "particle": {
+                            "icon_url": ":str:https://www.radixdlt.com/wp-content/uploads/2019/04/radix-logo.svg",
```

Comment:
should check with marketing if that's really the logo we want to use here
maybe it should be a permanent link like radixdlt.com/icon where there's always the most recent version

Reply:
That for sure isn't the icon that we want to use going forward.  I have an issue in the backlog to update this once we have a specific icon and location we want to use.